docs/planning/sprintplanning.md

```diff
@@ -0,0 +1,67 @@
+  >- 03/20 (Wednesday): All features and issues that fit the planning criteria are fed into voting document.
+  >- 03/27 (Wednesday): Pick top voted features and add them to 0.6 release based on estimates.
+
+> In future, we will need some kind of estimates for each feature for release planning. Shall we adopt story points on fibonacci scale or t-shirt sizing? My vote goes to story points on fibonacci scale to avoid spending a lot of time on estimation, which is a common pitfall.
+- Accepted features will be added to the correct milestone and sprint specific project for tracking.
+
+## How to manage Epic/Scenario?
+- Epic/Scenario typically spans 1-3 months or even more in some cases.
+- For new proposals these will be created and triaged/refined just like Story/Feature. However, after triaging these need to be broken down into Features for planning and voting.
+- Github doesn’t allow nested Issues, hence the only good way to track these are by creating Projects and adding the broken down features as issues to this project. An issue can be part of multiple projects and hence we can get a Scenario\Epic view as well as a Sprint view through respective projects.
+
+## Planning criteria:
+- Features are well defined and scoped with clear exit criteria. No feature creeps once release starts
+- Features can be implemented in <2w by one engineer. In case of Epic/Scenario, it could be broken down into <2w features with clear exit criteria.
+- The engineering team should be confident that any engineer can implement the feature based on the content in the feature *(This is similar to refinement in a scrum, but doesn’t have to be as strict as a functional specification)*.
+
+## Organizing with Github Projects 
+- New Project per sprint. Today, the sprint and milestone are both aligned. However in future we may decide to have multiple sprints in a release cycle.
+- New Project per any Story/Scenario
+- Any other projects that help organize and will be created when needed.
+
+## Organizing with Github Milestone 
+- A new milestone is created in Github for each 6 weeks release cycle.
+- All features and bugs that are planned for the release cycle are assigned to the milestone for tracking the progress of the release.
+
+## Pending work and proposed changes:
+- Decide estimation technique (story points on fibonacci, t-shirt size etc) - Postponed
+- How to automate project boards with Prow and issue labels?
+- Create features and bugs for 0.6 release.
+- Create appropriate templates - (In progress)
+- Create following labels, if they do not exist (in-progress)
+    - kind/feature-request
+    - kind/bug
+    - priority/critical-urgent
+      - Bugs that need to be fixed in current release, irrespective of when they are created.
+    - priority/important-soon
+      - Bugs that need to be fixed in the next release.
+    - priority/important-longterm
+     - Bugs that need to be fixed in next 2-3 releases..
+    - priority/awaiting-more-evidence
```

Comment:
Strongly suggest adding a label for triage similar to the one in build-pipeline (tekton) called [`meaty-juicy-coding-work`](https://github.com/tektoncd/pipeline/issues?q=is%3Aissue+is%3Aopen+label%3Ameaty-juicy-coding-work  )